content/en/user-guide/aws/lambda/index.md

```diff
@@ -232,6 +232,15 @@ However, the new provider still supports the following configuration options:
 * The `LAMBDA_DOCKER_DNS` option since LocalStack 2.2.
 * The `HOSTNAME_FROM_LAMBDA` option since LocalStack 3.0.
 
+## Lambda Event Source Mapping
```

Comment:
Consider changing the order:
<img width="350" alt="Screenshot 2024-01-04 at 19 15 14" src="https://github.com/localstack/docs/assets/8161292/8ad9ee21-3f66-474b-b607-cc345ff7e604">

a) Event Source Mapping is an AWS feature and should probably after "Getting Started" rather than at the bottom
b) "Special Tools" might fit better before the RB so we cover the AWS features first and then provide the LocalStack-specific added value like special tools and RB.